cmd/pr-reminder/main.go

```diff
@@ -77,6 +77,13 @@ func getInterestedLabels() sets.String {
 	return labels
 }
 
+// getUnactionablePrLabels returns a set of those labels that mark a PR which can't be reviewed in its current state
+func getUnactionablePrLabels() sets.String {
```

Comment:
I don't think that we need a new function for that. You can specify those as vars or constants.

Reply:
We could, I did it like this so the style is consistent with the `getInterestedLabels()` func above, and because it's convenient to have all the labels we need in one set I can work with later on. But I can change it.